django/contrib/postgres/forms/array.py

```diff
@@ -91,6 +91,18 @@ def run_validators(self, value):
         if errors:
             raise ValidationError(errors)
 
+    def has_changed(self, initial, data):
+        try:
+            data_value = self.to_python(data)
+        except ValidationError:
+            pass
```

Comment:
Test coverage is missing for this branch. You might need to add some initial test coverage for `has_changed()` in a separate commit.

Reply:
added testcase for existing implementation